src/api/java/com/github/minecraftschurlimods/arsmagicalegacy/api/spell/ISpellIngredient.java

```diff
@@ -14,40 +14,39 @@
  * Interface representing an ingredient for spell crafting.
  */
 public interface ISpellIngredient {
-    /**
-     * The codec used to serialize and deserialize any ingredient.
-     */
     Codec<ISpellIngredient> CODEC = ResourceLocation.CODEC.dispatch("type", ISpellIngredient::getType, ArsMagicaAPI.get().getSpellDataManager()::getSpellIngredientCodec);
-
-    /**
-     * The codec used to serialize and deserialize any ingredient in a network context.
-     */
     Codec<ISpellIngredient> NETWORK_CODEC = ResourceLocation.CODEC.dispatch("type", ISpellIngredient::getType, ArsMagicaAPI.get().getSpellDataManager()::getSpellIngredientNetworkCodec);
```

Comment:
Why remove the comments from the public constants?

Reply:
Because the usage is obvious, and we have a vast amount of network codecs that all don't have a comment (consistency!!1)